plugin/rt_scan_build_info_promote.go

```diff
@@ -0,0 +1,86 @@
+package plugin
+
+import (
+	"errors"
+	"log"
+)
+
+func GetScanCommandArgs(args Args) ([][]string, error) {
+	var cmdList [][]string
+
+	if args.BuildName == "" || args.BuildNumber == "" {
+		return cmdList, errors.New("Valid BuildName and BuildNumber are required")
+	}
+
+	tmpServerId := "tmpServeId"
+	jfrogConfigAddConfigCommandArgs, err := GetConfigAddConfigCommandArgs(tmpServerId,
+		args.Username, args.Password, args.URL, args.AccessToken, args.APIKey)
```

Comment:
Same here - created a reusable helper like `AddJfrogConfig` and call it wherever needed to reduce redundancy.
```
func AddJfrogConfig(args Args) ([]string, error) {
    return GetConfigAddConfigCommandArgs(tmpServerId, args.Username, args.Password, args.URL, args.AccessToken, args.APIKey)
}
```